src/main/groovy/cz/alenkacz/gradle/marathon/deploy/MarathonJsonEnvelope.groovy

```diff
@@ -8,11 +8,16 @@ import org.gradle.api.logging.Logger
 class MarathonJsonEnvelope {
     protected def Object parsedJson
     private PluginExtension pluginExtension
+    private BigDecimal mesosResourcesRatio
     private Logger logger
 
-    public MarathonJsonEnvelope(PluginExtension pluginExtension) {
+    public MarathonJsonEnvelope(PluginExtension pluginExtension, BigDecimal mesosResourcesRatio = 1.0) {
```

Comment:
What do you propose instead? We wanted to avoid rounding error of float. If there is any other Decimal type, it might be enough.